utils/warnings.ml

```diff
@@ -1207,7 +1207,7 @@ let message = function
   | Bs_fragile_external s ->
       s ^ " : the external name is inferred from val name is unsafe from refactoring when changing value name"
   | Bs_unimplemented_primitive s ->
-      "Unimplemented primitive used:" ^ s
+      "Unimplemented primitive used:" ^ s ^ ", make sure your library is using the `melange.ppx` preprocessor"
```

Comment:
make sure you are using `melange.ppx` right?

It can be not added in a melange.emit

Reply:
Good point. Update + included a link to migration docs.

Reply:
<3